package.json

```
@@ -24,8 +24,9 @@
     "@actions/exec": "^1.1.1",
     "@actions/github": "^5.1.1",
     "@zendesk/zcli": "^1.0.0-beta.16",
-    "axios": "^0.26.0",
-    "adm-zip": "^0.5.9",
+    "adm-zip": "^0.5.10",
+    "axios": "^1.4.0",
+    "jszip": "^3.10.1",
```

Essa lib não está sendo utilizada

```suggestion
    "axios": "^1.4.0"
```